config/main.yml

```diff
@@ -51,4 +51,7 @@ filterFeeds:
 versionFeeds:
   - project: MC
     channel: '666349583227682819'
-    scope: 5
\ No newline at end of file
+    scope: 5
+    actions:
+      - released
+      - renamed
```

Comment:
Might be worth adding "unreleased" too (as, if that ever happens, we would end up sending the release message twice, making it look like a bug)

Reply:
Hmm, yeah, that has never happened before, but probably worth adding anyway in case something goes wrong.